pkg/controller/garbagecollector/graph.go

```diff
@@ -67,10 +72,10 @@ func (n *node) isBeingDeleted() bool {
 	return n.beingDeleted
 }
 
-func (n *node) setDeletingDependents(v bool) {
+func (n *node) setDeletingDependentsToTrue() {
```

Comment:
I didn't name it "beginXXX" because that sounds like the GC is initiating the deletion of the object, while actually gc is only reflecting on the state changes received from the API server.

I'll change them to "markXXX".